guides/source/active_storage_overview.md

```diff
@@ -662,6 +662,35 @@ are stored before the form is submitted, they can be used to retain uploads when
 <%= form.file_field :avatar, direct_upload: true %>
 ```
 
+## Querying
+
+Active Storage attachments are just ordinary Active Record associations behind the scenes, so you can use the usual `joins` / `where` toolbox to look up records whose attachments meet specific criteria.
+
+### `has_one_attached`
+
+`has_one_attached` creates a `has_one` association called `"<name>_attachment"` and a `has_one :through` association called `"<name>_blob"`.
```

Comment:
```suggestion
`has_one_attached` creates a `has_one` association named `"<name>_attachment"` and a `has_one :through` association named `"<name>_blob"`.
```

Reply:
How about adding links to the `has_*_attached` methods?


```suggestion
[`has_one_attached`](https://api.rubyonrails.org/classes/ActiveStorage/Attached/Model.html#method-i-has_one_attached) creates a `has_one` association called `"<name>_attachment"` and a `has_one :through` association called `"<name>_blob"`.
```